redis/spring-cloud-starter-stream-sink-redis/src/main/java/org/springframework/cloud/stream/app/redis/sink/RedisSinkProperties.java

```diff
@@ -79,51 +79,63 @@ public void setTopic(String topic) {
 		this.topic = topic;
 	}
 
-	public Expression getKeyExpression() {
-		return key != null ? new LiteralExpression(key) : parser.parseExpression(keyExpression);
+	public Expression keyExpression() {
```

Comment:
Why isn't getter anymore?
Yes, it works like that, but it isn't Java Bean specification.